src/app/version-handler.ts

```diff
@@ -140,8 +142,22 @@ class VersionHandler {
 
             request.end();
 
+            logger.info('version-handler: mainUrl: ' + mainUrl);
+            logger.info('version-handler: hostname: ' + hostname);
+
             /* Get SFE version */
-            const urlSfeVersion = `${protocol}//${hostname}/client/version.json`;
+            let urlSfeVersion;
+            if (mainUrl?.includes('/client-bff/')) {
```

Comment:
I don't think this works for local SFE-Lite clients, since they don't build urls in the same way as static. Kiran created an API for checking if you use mana or not on SFE-Lite, can you use that?

Reply:
Agree with @psjostrom here. It also won't work if the deployment path changes from `client-bff` to something else in the future. I think we should use @KiranNiranjan's new API to find out if we are running Mana or not.

Reply:
![image](https://user-images.githubusercontent.com/55975938/100714939-c52c2080-33b6-11eb-9587-50617f1c03b8.png)

Reply:
Haha, in that case it all comes down to if we can live with not getting the correct values when we run local.